src/components/DatePicker/CalendarHeader/__tests__/CalendarHeader.spec.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
-import toJson from 'enzyme-to-json';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
 import { CalendarHeader } from '../CalendarHeader';
 
 describe('CalendarHeader', () => {
-    const YEARS = [2016, 2025];
+    const YEARS = [2016, 2025] as [number, number];
     const decreaseMonth = jest.fn();
     const increaseMonth = jest.fn();
     const changeYear = jest.fn();
-    let wrapper;
+    let view;
 
     beforeEach(() => {
-        wrapper = mount(
+        view = render(
+            // @ts-ignore
```

Comment:
why // ts-ignore?

Reply:
It asks to provide all properties:
TS2740: Type '{ yearsRange: [number, number]; date: Date; decreaseMonth: Mock ; increaseMonth: Mock ; changeYear: Mock ; }' is missing the following properties from type 'Props': monthDate, changeMonth, customHeaderCount, prevMonthButtonDisabled, and 5 more.

Reply:
What is the reason for not providing them all?

Reply:
:+1: